[PLT-1917][PLT-1968][PLT-1971][PLT-1957][PLT-1956][PLT-1958] Improve upgrade flow (#740)
Stratio/kind: stratio-docs/en/modules/operations-manual/pages/offline-installation.adoc

* _ClusterConfig_:
** *`spec.private_registry`: _true_*. With this parameter, you tell _Stratio Cloud Provisioner_ that all images needed during provisioning and installation of the cluster should be retrieved from the _keos registry_.

> ** *`spec.private_helm_repo`: _true_*. With this parameter, you tell _Stratio Cloud Provisioner_ that all charts needed during provisioning and installation of the cluster should be retrieved from the _keos helm repository_.

Reviewer: ```suggestion
** *`spec.private_helm_repo`: _true_*. This parameter tells _Stratio Cloud Provisioner_ that all the charts required for provisioning and installing the cluster should be fetched from the _keos registry_.
```

* _KeosCluster_:

> ** `spec.docker_registries`: in the 'docker_registries' section, the _keos registry_ from which the images will be downloaded for offline deployment must be indicated. This registry must have the necessary images for each installation.

Reviewer: ```suggestion
** `spec.docker_registries`: in the 'docker++_++registries' section, the _keos registry_ from which the images will be downloaded for offline deployment must be indicated. This registry must have the necessary images for each installation.
```
